app/assets/javascripts/va_facilities/facility_created_practice_search.es6

```diff
@@ -105,13 +98,19 @@ function setDataAndMakeRequest({ isNextPage = false, searchTermInput }) {
     ? parseInt($(cp.categoriesInput).parent().find(".usa-select")[0].value)
     : null;
 
-  let searchTerm =
-    $(cp.searchField).data("search").length > 0
-      ? $(cp.searchField).data("search")
-      : null;
+    let createdSearchField = document.getElementById('dm-created-practice-search-field');
+    let searchTerm = null;
+    if (createdSearchField !== null){
```

Comment:
Same here. I think we can use `if (createdSearchField)`

Reply:
Got it.